src/Plugin/Drupal7/Audit/Database/DatabaseConnectionAnalysis.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+namespace Drutiny\Plugin\Drupal7\Audit;
+
+use Drutiny\Audit\AbstractAnalysis;
+use Drutiny\Sandbox\Sandbox;
+
+/**
+ * Provides all database connection information.
+ */
+class DatabaseConnectionAnalysis extends AbstractAnalysis {
+
+  public function gather(Sandbox $sandbox) {
+    $db_connections = $this->target->getService('drush')
+      ->runtime(function() {
+        global $databases;
+        return $databases;
```

Comment:
I think we should copy $databases and remove credentials on that copy before returning it. That means the removed credentials never leave the Drupal runtime environment.

Reply:
Done.